js/spotify.js

```diff
@@ -1,12 +1,48 @@
-// API Docs at: 
+// API Docs at:
 // https://developer.spotify.com/web-api/search-item/
+$(document).ready(function () {
 
+  $('#search-type artist').on('click', function(e) {
+    var artists = $(this).text();
 
-function searchByArtist(keyword) {
-  var url = 'https://api.spotify.com/v1/search?q='+keyword+'&type=artist';
-}
+    searchByArtist(artists);
+    e.preventDefault();
+  })
+  searchByArtist('#results');
 
+  function searchByArtist(keyword) {
```

Comment:
This function doesn't have to be defined inside the doc.ready. It can be defined outside of it and then invoked inside with the event listener.